.github/workflows/integration-test-cassandra-suite.yaml

```
@@ -0,0 +1,121 @@
+name: Integration test CassandraSuite
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+  pull_request:
+
+jobs:
+  TestCassandraSuite:
+    if: ${{ github.event_name == 'pull_request' && github.ref != 'refs/heads/master' }}
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes-versions : ['v1.15.12', 'v1.20.2']
```

Other tests were just updated to 1.21.0
```suggestion
        kubernetes-versions : ['v1.15.12', 'v1.21.0']
```